app/experimenter/experiments/models/nimbus.py

```diff
@@ -195,6 +195,24 @@ def __str__(self):
         return self.name
 
 
+class NimbusDocumentationLink(models.Model):
+    experiment = models.ForeignKey(
+        NimbusExperiment,
+        related_name="documentation_links",
+        on_delete=models.CASCADE,
+    )
+    title = models.CharField(max_length=255, null=False)
+    link = models.URLField(max_length=255, null=False)
+
+    class Meta:
```

Comment:
We could put a unique_together here on `('title', 'experiment')` to prevent duplicated links on a single experiment.